spec/requests/sessions_spec.rb

```diff
@@ -0,0 +1,152 @@
+# frozen_string_literal: true
+
+describe 'Sessions', type: :request do
+  let(:recaptcha_id) { 'recaptcha-form' }
+
+  describe '#new' do
+    after { cookies[:login_recaptcha] = nil }
+
+    context 'when recaptcha is not configured' do
+      before do
+        allow(RecaptchaService).to receive(:recaptcha_configured?).and_return(false)
+      end
+
+      it 'does not display recaptcha' do
+        cookies[:login_recaptcha] = true
+        get new_user_session_path
+
+        expect(response.body).to_not include recaptcha_id
```

Comment:
This doesn't feel great, but I wasn't quite sure how to verify that the recaptcha loaded on the page. I'm open to any suggestions!

Reply:
Seems good enough for a request spec to me.